keps/sig-node/3675-resource-plugin-manager/README.md

```diff
@@ -0,0 +1,920 @@
+    # 6 exclusive cores(no other processes running on them)
+    # 4 shared cores(other processes can run on them) 
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: example-compute-claim-parameters
+      namespace: default
+    data:
+      cores: |
+    # number of core type 1
+    2
+    # number of core type 2
+    6
+    # number of core type 3
+    4
+      compute_attributes: |
+        # Attributes of core type 1
+        exclusive, smt-sibling-required, 2.1Ghz, 1.0 priority, no-irq
+        # Attributes of core type 2
+        exclusive
+        # Attributes of core type 3
+        shared
+
+##### Example 2.: Policy-based resource specification
+
+Another option how a resource allocation configuration for Kubernetes
+Pods could be achieved is through a policy mechanism. In this case the
+policy will be applied to all Pods to be processed. The configuration
+of the policy can happen directly in a CCI Compute Resource Driver Plugin
+as input argument in the yaml deployment. This methodology can be used to
+realize methodologies similar to the static CPU manager policy or other policies.
+
+##### Resource Manager Architecture
+
+The extension consists of an optional CCI driver inside Pod spec which
+indicates if a CCI driver exists, then it shall process compute resources.
+The association betweed pod and CCI Driver can be also realised as a resource class 
+argument (will be considered as an option during the implementation). 
+
+In the alpha version of the Resource Manager applies driver plugins to manage resource
+allocations for Pods. Post-alpha we will consider if we need to explicitly mark which pods are to be handled by an allocation 
+plugin.  The overall architecture consists of a manager component – the Resource
+Manager inside the container manager, an optional Pod spec argument to identify which 
+Pods to be processed by the manager(post-alpha), a compute resource store – a component used to store
+the resource sets(cpusets, memory affinity configuration) for the resource plugins. The CCI Manager 
+is enabled via a feature gate. After turning on the feature gate all pod allocation requets for cpu 
+and memory will be handled by the CCI Manager.
+
+![image](CCIArchAlpha.jpg)<br>
+Figure 1: CCI Resource Manager Architecture inside Kubelet
+
+    Pod Spec “CCIDriverName Extension”
+    type PodSpec {
+       …
+    	 // CCIResourceDriver is an optional parameter which can identify if a Pod resource manager (cpu an memory)
+	    // has to be handled by a pluggable CCI resource driver 
+	    // This is an alpha field and requires enabling the
+    	 // CCIResourceManager feature gate.
+	    //
+    	 // This field is immutable.
+	    //
+	    // +featureGate=CCIResourceManager
+	    // +optional
+	    CCIResourceDriver string `js”n:”cciResourceDriver,omitem”ty” protob”f:”bytes,40,opt,name=cciResourceDri”er”`
+     …}
+
+The proposed extension relies on a new optional argument inside the Pod spec to `driverName`. 
```

Comment:
Above we have specified that the field would be `cciResourceDriver`? Please use consistent naming.

Reply:
done